wdfn-server/waterdata/views.py

```diff
@@ -191,6 +191,28 @@ def hydrological_unit_locations(huc_cd):
     return hydrological_unit(huc_cd, show_locations=True)
 
 
+@app.route('/networks/', defaults={'network_cd': 'ALL'}, methods=['GET'])
+@app.route('/networks/<network_cd>/', methods=['GET'])
+@defined_when(app.config['NETWORK_PAGES_ENABLED'], return_404)
+def networks(network_cd):
+    """
+    Network unit view
+    :param network_cd: ID for this network
+    """
+
+    # Grab the Network info
+    collection = ogc.get_networks(network_cd)
+
+    http_code = 200 if (network_cd != '') else 404
+
+    return render_template(
+        'networks.html',
+        http_code=http_code,
```

Comment:
I don't think we should be passing the http_code as part of the context data. The Flask framework handles non 200 http codes automatically.